projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-posts.php

```diff
@@ -27,6 +27,23 @@ public function setUp() {
 		$this->sender->do_sync();
 	}
 
+	/**
+	 * Verify post_content is limited based on MAX_POST_CONTENT_LENGTH.
+	 */
+	public function test_post_content_limit() {
+
+		$post_sync_module = Modules::get_module( 'posts' );
+
+		$this->post->post_content = str_pad( '', Automattic\Jetpack\Sync\Modules\Posts::MAX_POST_CONTENT_LENGTH - 1, 'X', STR_PAD_LEFT );
```

Comment:
This works, but it seems weird. It would be more straightforward to use `str_repeat`.
```suggestion
		$this->post->post_content = str_repeat( 'X', Automattic\Jetpack\Sync\Modules\Posts::MAX_POST_CONTENT_LENGTH - 1 );
```